src/content/reference/developer-tooling/react-performance-tracks.md

```diff
@@ -0,0 +1,137 @@
+---
+title: React Performance tracks
+version: experimental
```

Comment:
We can mention canary here and in other places now with https://github.com/facebook/react/pull/34665

Reply:
Updated to "Canary".

Will update to stable with the next stable release.